runtime/common/address.go

```diff
@@ -61,6 +61,12 @@ func (a Address) Hex() string {
 	return fmt.Sprintf("%x", a[:])
 }
 
+// HexInto encodes the address as a hex string representation of the address
+// into the given target byte slice.
+func (a Address) HexInto(target []byte) {
```

Comment:
This naming is less confusing for me:

```suggestion
// ToHex encodes the address as a hex string representation of the address
// into the given target byte slice.
func (a Address) ToHex(target []byte) {
```

Reply:
This wasn't actually needed, so I removed it